webapp/src/main/webapp/themes/nemo/templates/head.ftl

```diff
@@ -20,6 +20,8 @@
     </#if>	
 </#if>	
 
+<script src="${urls.base}/bootstrap-5.3.2/js/bootstrap.min.js"></script> 
```

Comment:
It seems bootstrap5 and bootstrap3 would be loaded at the same time according to head.ftl and headScripts.ftl of Nemo theme.